src/charm.py

```diff
@@ -205,6 +187,74 @@ def _check_kf_profiles(self, interfaces):
 
         return kf_profiles
 
+    def _create_resources(self, resource_type: List[str] = None) -> None:
```

Comment:
I think a lot of what is done in this overlaps with what we're doing in the [chisme KubernetesResourceHandler](https://github.com/canonical/charmed-kubeflow-chisme/blob/a0a99c9675c0a9948558fb957a18fb601762f67f/src/charmed_kubeflow_chisme/kubernetes/_kubernetes_resource_handler.py#L37).  Have you seen that helper?  If not, take a look.  If it doesn't make sense for the use here, can you explain what doesn't work?

Reply:
Firstly I tried ... apply many is not patching resources if failed .... I was thinking to update chisme but it could be breaking change for that function ....